TW 308 API governance section (#4162)
postmanlabs/postman-docs: src/pages/docs/api-governance/api-governance-overview.md

---
title: "API governance"
updated: 2022-07-20
search_keyword: "api security, api schema, security warnings, schema validation, security validation, api security audit, api security scan, api schema vulnerabilities, security audit"
---

> _API governance_ is the practice of applying standards consistently across your API design and testing phases of your development process. This consistency reduces friction and streamlines collaboration across all parts of the API development lifecycle.

Author: Updated in [a8b1c04](https://github.com/postmanlabs/postman-docs/pull/4162/commits/a8b1c042aecd096666f1f06a82c642dee04f3016). The new intro paragraph reads:

`_API governance_ is the practice of applying a defined set of standards consistently across your API design and testing phases of your development process. Applying standards consistently ensures that you can develop quality APIs more quickly than you could if you do not enforce API governance and streamlines collaboration across all parts of the API development lifecycle.`

@savageCoder55 Lmk if this works!

> _API governance_ is the practice of applying a defined set of standards consistently across your API design and testing phases of your development process. Applying standards consistently ensures that you can develop quality APIs more quickly than you could if you do not enforce API governance and streamlines collaboration across all parts of the API development lifecycle.

Reviewer: ```suggestion
_API governance_ is the practice of applying a defined set of standards consistently across your API design and testing phases of your development process. Applying standards consistently helps you to develop high-quality APIs and streamlines collaboration across all parts of the API development lifecycle.
```

> _API governance_ is the practice of applying a defined set of standards consistently across your API design and testing phases of your development process. Applying standards consistently ensures that you can develop quality APIs more quickly than you could if you do not enforce API governance and streamlines collaboration across all parts of the API development lifecycle.

Author: ```suggestion
_API governance_ is the practice of applying a defined set of standards consistently across your API design and testing phases of your development process. Applying standards consistently helps you to develop high-quality APIs and streamlines collaboration across all parts of the API development lifecycle.
```

> _API governance_ is the practice of applying a defined set of standards consistently across your API design and testing phases of your development process. Applying standards consistently ensures that you can develop quality APIs more quickly than you could if you don't enforce API governance. API governance streamlines collaboration across all parts of the API development lifecycle.

Author: ```suggestion
_API governance_ is the practice of applying a defined set of standards consistently across your API design and testing phases of your development process. Applying standards consistently helps you to develop high-quality APIs and streamlines collaboration across all parts of the API development lifecycle.
```

> Postman’s API governance features offer you guidance for APIs at the design and testing phases. By identifying possible inconsistencies or weak points and pointing out possible fixes or improvements, Postman enables you to create secure APIs that follow industry best-practices.

Reviewer 2: Couple tweaks that I think will help : 

"guidance for APIs at the design, develop and testing phases"

"Postman enables you to create better and more secure APIs..."

Author: Thanks - added.
